README.md

```diff
@@ -1,19 +1,24 @@
 
-The [Rinkeby DAO](https://rinkeby.aragon.org/#/flight.aragonid.eth/), recently created, will be used to test this proposed ACGP process and have the community signalling about desired decisions about new membership process, management of the AragonCoop operations, processes and funds.
-After having a first feedback from the community, we shall jointly decide which kind of decisions shall be migrated to the Mainnet DAO to consider them as completely binding or if we can consider the Rinkeby decisions as binding without going to mainnet.
+The [RinkebyDAO](https://rinkeby.aragon.org/#/flight.aragonid.eth/), recently created and purposefully unstable, will be made use of to explore a variety of DAO tools, including [That Planning Suite](https://github.com/AutarkLabs/planning-suite), as well as these proposed ACGP processes. We intend for the CGP's to provide a place for community signalling discussions and decisions.
+
+He ACGP proposer assigns each vote either a ``Stick``: where the ACGP is enacted on Rinkeby (for more 'trivial' proposals), or a ``Twist``;( for more 'serious' proposals, i.e. as the above (line 12) 'binding changes to ratied processes'). After initial feedback and voting from the community, 'serious' votes should be migrated to the MainnetDAO (in less than one month). MainnetDAO votes should run no more than once (1) per month, with less than ten (<=10) votes maximum, in total, per month.
+
+Community discussions around all aspects of the AragonCooperativeDAO are active on keybase/aragoncoop.
 
 ## How do I propose a new ACGP?
 
-It is as easy as cloning/copying the [ACGPtemplate.md](./ACGPtemplate.md) file and creating a new file with your proposal in the [ACGPs](./ACGPs/) folder in this Github repository and create yourself (you need to have a membership token of course) a new vote in the [Rinkeby DAO](https://rinkeby.aragon.org/#/flight.aragonid.eth/) once your AGP is in the master branch of this repository.
+First off you require some MRT Tokens - ten (10) to be precise. MRT (Membership Reputation Tokens) are earned by transacting with the AragonCooperativeDAO. THe cost of proposing an ACGP is ten (10) MRT Tokens (to be burned). It is then as easy as cloning/copying the [ACGPtemplate.md](./ACGPtemplate.md) file and creating a new file with your proposal in the [ACGPs](./ACGPs/) folder in this Github repository (please get in touch if any issues). Once your AGP is in the master branch of this repository, follow the instructions to create a new vote in the [Rinkeby DAO](https://rinkeby.aragon.org/#/flight.aragonid.eth/).
+
+The cost of voting in an ACGP is one (1) MRT Token (to be burned) - you are expected to take part in ACGP discussions (& earn MRT for doing so, 1hr = 1MRT).
```

Comment:
Since there is currently no role for voting in the voting app, this would require either a custom voting app, or a custom forwarder that implemented the MRT burning.